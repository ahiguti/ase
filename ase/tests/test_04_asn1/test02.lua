
local Test = ASE.CPP['ASETestCPP']
local arr = { 10, "foobar", 99.99999 }
local s = Test.ToXML(arr)
local v = Test.FromXML(s);
for i = 0, v.Length - 1 do
  print("v " .. i .. " " .. v[i])
end
