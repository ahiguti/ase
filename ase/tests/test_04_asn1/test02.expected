v 0 10
v 1 foobar
v 2 99.99999
